src/main/java/plugily/projects/villagedefense/arena/managers/ZombieSpawnManager.java

```diff
@@ -131,26 +126,22 @@ public void spawnZombies() {
       spawn = (int) Math.ceil(zombiesLimit / 2.0);
     }
 
-    if(arena.getZombies().isEmpty()) {
-      for(int i = 0; i <= spawn; i++) {
-        if(arena.getOption(ArenaOption.ZOMBIES_TO_SPAWN) > 0) {
-          arena.spawnFastZombie(random);
-        }
-      }
-    }
+//    if(arena.getZombies().isEmpty()) {
+//      for(int i = 0; i <= spawn; i++) {
+//        if(arena.getOption(ArenaOption.ZOMBIES_TO_SPAWN) > 0) {
+//          arena.spawnFastZombie(random);
```

Comment:
...and where the fast zombies will be spawned?

Reply:
This is just a draft. Wait for the full PR. I have just converted FastZombie, not others.

Reply:
There will be a manager for zombie spawners. Be patient.